From fc89fb21616929139e555955515d47c26851bb3b Mon Sep 17 00:00:00 2001
From: Jason Smith <jsmith@habana.ai>
Date: Thu, 29 May 2025 09:18:35 -0400
Subject: [PATCH] [SW-230673] pldm: remove deprecated kconfig

All firmware has been migrated to use ASCII aux names; remove
deprecated kconfig used for the transition.

Change-Id: Ib99a699768cb7271f4183bfd61bf61b237b4638f
Signed-off-by: Jason Smith <jsmith@habana.ai>
---

diff --git a/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig b/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig
index 547f93b..ffc5c8d 100644
--- a/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig
+++ b/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig
@@ -82,7 +82,6 @@
 CONFIG_LIBPLDM_MAX_MSG_SIZE=1024
 CONFIG_LIBPLDM_RECV_TIMEOUT_MS=10
 CONFIG_LIBPLDM_FIRMWARE_UPDATE=y
-CONFIG_LIBPLDM_PDR_ASCII_AUX_NAMES=y
 CONFIG_BOOTINFO_OOBI_SUPPORT=y
 CONFIG_HL_RESET_DRIVER=y
 CONFIG_THREAD_NAME=y
diff --git a/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig b/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
index a5fd7d5..a0f1092 100644
--- a/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
+++ b/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
@@ -129,7 +129,6 @@
 CONFIG_LIBPLDM_PDR_OFFLOAD=y
 CONFIG_SENSORD_USE_MEM_POOL_FOR_STATE=y
 CONFIG_BOOTINFO_OOBI_SUPPORT=y
-CONFIG_LIBPLDM_PDR_ASCII_AUX_NAMES=y
 CONFIG_ASC10=y
 CONFIG_TMP464=y
 CONFIG_TMP432=y
diff --git a/boards/hl/gaudi3/hl_gaudi3_preboot_asic_defconfig b/boards/hl/gaudi3/hl_gaudi3_preboot_asic_defconfig
index 4f3144a..f5e9735 100644
--- a/boards/hl/gaudi3/hl_gaudi3_preboot_asic_defconfig
+++ b/boards/hl/gaudi3/hl_gaudi3_preboot_asic_defconfig
@@ -82,7 +82,6 @@
 CONFIG_LIBPLDM_MAX_MSG_SIZE=1024
 CONFIG_LIBPLDM_RECV_TIMEOUT_MS=10
 CONFIG_LIBPLDM_FIRMWARE_UPDATE=y
-CONFIG_LIBPLDM_PDR_ASCII_AUX_NAMES=y
 CONFIG_BOOTINFO_OOBI_SUPPORT=y
 CONFIG_EEPROM=y
 CONFIG_LIBEEPROM=y
diff --git a/modules/Kconfig.libpldm b/modules/Kconfig.libpldm
index bbb6f23..1c8917e 100644
--- a/modules/Kconfig.libpldm
+++ b/modules/Kconfig.libpldm
@@ -32,11 +32,6 @@
 	depends on LIBPLDM
 	default n
 
-config LIBPLDM_PDR_ASCII_AUX_NAMES
-	bool "PLDM aux names enable ascii encoding"
-	depends on LIBPLDM
-	default n
-
 config LIBPLDM_TEST
 	bool "PLDM Library Testing Support"
 	depends on LIBPLDM
